src/durabletask/azext_durabletask/aaz/latest/durabletask/scheduler/_create.py

```diff
@@ -49,7 +49,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
             required=True,
         )
         _args_schema.scheduler_name = AAZStrArg(
-            options=["-n", "--name", "--scheduler-name"],
+            options=["-n", "--scheduler-name"],
```

Comment:
What's motivating the removal of `--name`?

Reply:
This was a mistake and it will be added back in